cli/print/clusters.go

```diff
@@ -15,19 +16,54 @@ var clustersTmplSrc = `ID	NAME	K8S DISTRO	K8S VERSION	STATUS	CREATED	EXPIRES
 
 var clustersTmpl = template.Must(template.New("clusters").Funcs(funcs).Parse(clustersTmplSrc))
 
-func Clusters(w *tabwriter.Writer, clusters []*types.Cluster) error {
-	if err := clustersTmpl.Execute(w, clusters); err != nil {
+func Clusters(outputFormat string, w *tabwriter.Writer, clusters []*types.Cluster) error {
+	if outputFormat == "table" {
+		if err := clustersTmpl.Execute(w, clusters); err != nil {
+			return err
+		}
+		return w.Flush()
+
+	}
+	if outputFormat == "json" {
+		defer w.Flush()
+		var cAsByte []byte
+		cAsByte, _ = json.MarshalIndent(clusters, "", "  ")
+		_, err := w.Write(cAsByte)
 		return err
 	}
-
-	return w.Flush()
+	return nil
 }
 
-func NoClusters(w *tabwriter.Writer) error {
-	_, err := w.Write([]byte(`No clusters found. Use the "replicated cluster create" command to create a new cluster.`))
-	if err != nil {
+func NoClusters(outputFormat string, w *tabwriter.Writer) error {
+	if outputFormat == "table" {
+		_, err := w.Write([]byte(`No clusters found. Use the "replicated cluster create" command to create a new cluster.`))
+		if err != nil {
+			return err
+		}
+
+		return w.Flush()
+	} else if outputFormat == "json" {
+		defer w.Flush()
+		_, err := w.Write([]byte("[]"))
 		return err
 	}
+	return nil
+}
 
-	return w.Flush()
+func Cluster(outputFormat string, w *tabwriter.Writer, cluster *types.Cluster) error {
+	if outputFormat == "table" {
+		if err := clustersTmpl.Execute(w, []types.Cluster{*cluster}); err != nil {
+			return err
+		}
+		return w.Flush()
+
+	}
+	if outputFormat == "json" {
```

Comment:
Would be good to see some consistency in style.  Some of these functions use `if` and some `else if`.